recipe/build.sh

```diff
@@ -45,6 +45,7 @@ export CFLAGS="${CFLAGS} -fno-merge-constants"
 
 make -j${CPU_COUNT} ${VERBOSE_AT}
 make install
+alias touch=gtouch
```

Comment:
Why was this added? `touch` is a standard unix command. We shouldn't need to alias it to `gtouch`.

Reply:
gtouch was used instead of touch due to failures happening with the native touch command in the tests. @ELundby45 might be able to elaborate more on the why.

Reply:
`touch -d 0 test.test`

This works on most unix systems, but not in MacOS. Adding `coreutils` brings in the GNU compliant version of touch which is what the unit test is expecting. 

It probably makes sense to only set the alias for MacOS.

Reply:
I agree the alias should only be set for MacOS. And we probably can remove the patch, too.

Reply:
went ahead and added some code thanks to @ELundby45 to make it MacOS specific